src/Sylius/Component/Channel/spec/Model/ChannelSpec.php

```diff
@@ -66,13 +71,34 @@ function its_color_is_mutable()
 
     function it_is_enabled_by_default()
     {
-        $this->shouldBeEnabled();
+        $this->isEnabled()->shouldReturn(true);
```

Comment:
I think `$this->shouldBeEnabled()` and `$this->shouldNotBeEnabled()` is more readable.